.github/workflows/api-release.yml

```
@@ -39,0 +1,81 @@
+          image-tags: |
+            type=raw,value=${{ inputs.release-version }}
+            type=raw,value=latest,enable=${{ inputs.tag-latest == true }}
+
+  create-github-release:
+    if: ${{ !failure() && !cancelled() }}
+    needs: "build-image"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create GitHub Release
+        id: create_release
+        uses: softprops/action-gh-release@da05d552573ad5aba039eaac05058a918a7bf631 # v2.2.2
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)